troubleshooting.md

```diff
@@ -0,0 +1,551 @@
+# Troubleshooting Guide for BM IPI Deployments
+
+* [Bootstrap VM issues](#bootstrap-vm-related-issues)
+
+* [Following the Installation](#following-the-installation) 
+
+* [Node Related Issues](#node-related-issues)
+
+* [NTP out of sync](#ntp-out-of-sync)
+
+* [Registry Issues](#registry-issues) 
+
+* [IPv6 Disconnected Install Troubleshooting](#ipv6-disconnected-install-troubleshooting)
+
+* [Misc Issues](#misc-issues) 
+
+
+# Bootstrap VM related issues 
+
+* Cannot login to the Bootstrap VM
+```
+$ ssh core@172.22.0.2
+Permission denied (publickey,password,keyboard-interactive).
+```
+
+When attempting to access the Bootstrap VM and `permissioned_denied` error occurs, verify that your SSH key for the user attempting to log into the VM is set within your `install-config.yaml` file. 
+
+
+After the installation is triggered, Wait for few minutes and check/verify bootstrap VM is up and running from Provisioner node: 
```

Comment:
this needs better flow. it went from discussing permissioned denied to talking about bootstrap VM being triggered. needs better flow and context as to how you got to that sentence. likely a subheader too

Reply:
Added a proper flow to this process.